main/varnish/APKBUILD

```diff
@@ -20,7 +21,6 @@ pkggroups="varnish"
 source="http://varnish-cache.org/_downloads/$pkgname-$pkgver.tgz
 	musl-include-vpf.patch
 	musl-include-vsb.patch
-	musl-fix-stack-overflow.patch
```

Comment:
I need to remove the file as well

Reply:
yes. `git rm musl-fix-stack-overflow.patch && git commit --amend` and then `git push --force`